<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetMouseEvent</class>
 <widget class="QDialog" name="GetMouseEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add mouse event</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/RES/ffficons/mouse.png</normaloff>:/resources/RES/ffficons/mouse.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select mouse event:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Event type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="EventType">
       <item>
        <property name="text">
         <string>Clicked on this object</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Released on this object</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button pressed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button released</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button is down</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wheel up</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wheel down</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="BtnLabel">
       <property name="text">
        <string>Button:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="ButtonMenu">
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Middle</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GetMouseEvent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GetMouseEvent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
